en/V3-Software_Platform_Requirements.md

```diff
@@ -17,10 +17,10 @@ The Linux operating system is one of the most popular in IoT. It has many featur
 | **3.1.1** | Verify that the bootloader does not allow code to be loaded from arbitrary locations. Locations include both storage (SD, USB, etc.) and network locations (over TCP/IP). | ✓ | ✓ | ✓ |
 | **3.1.2** | Verify bootloader configurations are immutable in production releases. | | ✓ | ✓ |
 | **3.1.3** | Verify that communication interfaces such as, USB, UART, and other variants are disabled or adequately protected during every stage of the device's boot process. | | ✓ | ✓ |
-| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. e.g. CPU Based Secure Boot/Trusted Boot | | ✓ | ✓ |
-| **3.1.5** | Verify that the authenticity of next bootloader stages or application code is cryptographically verified during every step of the boot process. | | ✓ | ✓ |
+| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. e.g. CPU Based Secure Boot/Trusted Boot/Hardware Trust Anchor (HTA) | | ✓ | ✓ |
```

Comment:
Secure boot (or trusted boot) refers to the process of verifying the authenticity of the different boot stages. In order to be effective, we want the cryptographic secrets that are used to verify this authenticity to be secured by hardware. The way its phrased now, it seems like HTA is a form of secure boot. What about the following phrasing? 
`... e.g. Secure/trusted boot with a hardware root of trust`

Reply:
make sense, agree

Reply:
Added